source/less/blocks/offer.less

```
@@ -0,0 +1,35 @@
+.offer {
+  background-color: @color-grey;
+}
+
+.offer-tagline {
+  width: 100%;
+  min-height: 250px;
+  padding: 46px 20px 70px 20px;
+  box-sizing: border-box;
+
+  p {
+    font-size: 24px;
+    font-weight: 300;
+    line-height: 120%;
+  }
+}
+
+.offer-list {
+  min-height: 828px;
+  .ul_style ();
+  background-image: url("/img/phone_smile.jpg"), url("/img/grey_background.svg");  //как сделать, чтобы фон наезжал на другой? (Верх серого фона)
```

```suggestion
  background-image: url("../img/phone_smile.jpg"), url("../img/grey_background.svg");  //как сделать, чтобы фон наезжал на другой? (Верх серого фона)
```
На счёт вопроса: я не наблюдаю данных файлов в вашем проекте. Когда добавите, тогда смогу подсказать.